website/blog/2024-04-15-0.74-post.md

```diff
@@ -0,0 +1,407 @@
+---
+title: 'React Native 0.74'
+authors:
+  [hurali, alanjhughes, alfonsocj, ingridwang, huntie, cortinico]
+tags: [release]
+date: 2024-04-15
+---
+
+Today we're releasing React Native 0.74!...
+
+## Highlights
+
+<!-- https://github.com/facebook/react-native/blob/main/CHANGELOG.md -->
+
+- [React Native Frameworks](/blog/2024/04/15/0.74-post#react-native-frameworks)
+- New layout behaviors
+- Support for `align-content: 'space-evenly'`
+- New Architecture support for `position: 'static'`
+- [Preparing for the New Architecture?](/blog/2024/04/15/0.74-post#preparing-for-the-new-architecture)
+- [Batched `onLayout` updates (New Architecture)](/blog/2024/04/15/0.74-post#batched-onlayout-updates-new-architecture)
+- [Removal of deprecated `PropTypes`](/blog/2024/04/15/0.74-post#removal-of-deprecated-proptypes)
+- [Deprecated APIs in `PushNotificationIOS`](/blog/2024/04/15/0.74-post#deprecated-apis-in-pushnotificationios)
+- [Android SDK minimum bump (Android 6.0)](/blog/2024/04/15/0.74-post#android-sdk-minimum-bump-android-60)
+- [Yarn 3 is the default for new projects](/blog/2024/04/15/0.74-post#yarn-3-for-new-projects)
+
+<!--truncate-->
+
+## React Native Frameworks
+
+- https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0759-react-native-frameworks.md
```

Comment:
I think this part requires a couple of extra lines to explain in a couple of words what are we talking about, before redirecting the reader to the proposal if they want to learn more. What do you think?
cc. @cortinico which is probably the best person to draft these.

Reply:
@huntie I think this part should probably go entirely, as most of the announcements will go with 0.75